common/api/ui-components.api.md

```diff
@@ -4036,15 +4028,13 @@ export interface SolarDataProvider {
     day: Date;
     readonly dayStartMs: number;
     onTimeChanged?: SolarPlaybackProgressHandler;
-    setDateAndTime: (day: Date, isProjectDate?: boolean) => void;

```

Comment:
I just recently added setDateAndTime function, so must have a bad merge or need to rerun extract-api in ui-components.